prql-compiler/tests/integration-vendors/docker-compose.yml

```
@@ -0,0 +1,39 @@
+version: "3"
+
+services:
```

Note that `version` is not required in Compose Spec.

```suggestion
services:
```